.github/workflows/platform-pull-request.yml

```diff
@@ -36,10 +36,10 @@ jobs:
             - name: Run E2E tests against local
               uses: ./.github/actions/e2e-tests
               with:
-                e2e_test_token: some-token
-                slack_token: ${{ secrets.SLACK_TOKEN }}
-                environment: local
-                enable_influxdb_features: false
+                  e2e_test_token: some-token
```

Comment:
If we are using an IDE to apply space linting, I think we need to have rules defined so that other people's IDE will follow similarly.

I think https://editorconfig.org/ will allow for this.

Reply:
Can we use prettier to do this perhaps? We already have the `.prettierrc.json` in the project and I think it supports YAML.

Reply:
Yeah we just don't have linting on the yaml - I will see to add